Client/sdk/game/CWorld.h

```diff
@@ -61,6 +61,16 @@ struct SProcessLineOfSightMaterialInfoResult {
     bool        valid{};       //< Data found in this struct is only valid if this is `true`!
 };
 
+struct STestSphereAgainstWorldResult
+{
+    bool           collisionDetected{false};
+    std::uint32_t  modelID{0};
+    CVector        entityPosition{};
+    CVector        entityRotation{};
+    std::uint32_t  lodID{0};
+    std::uint8_t   type{0};
```

Comment:
you can use eEntityType here

Reply:
eEntityType is unavailable in this file. It’s probably better not to include CEntity.h here, right?

Reply:
Why?